parquet-column/src/main/java/org/apache/parquet/filter2/predicate/ValidTypeMap.java

```diff
@@ -74,9 +74,22 @@ private static void add(Class<?> c, FullTypeDescriptor f) {
     // Both of these binary types are valid
     add(Binary.class, new FullTypeDescriptor(PrimitiveTypeName.BINARY, null));
     add(Binary.class, new FullTypeDescriptor(PrimitiveTypeName.FIXED_LEN_BYTE_ARRAY, null));
+    add(Binary.class, new FullTypeDescriptor(PrimitiveTypeName.INT96, null));
```

Comment:
so, for something like this, does it make more sense to use a different java class other than Binary? we could use String for UTF8, and BigNum or something here?

Maybe that could be a higher level API on top of this one though.